tests/README.md

```diff
@@ -1,26 +1,84 @@
 ## Preparing K8S environment for tests
```

Comment:
Maybe we should mention that it's an instruction for Ubuntu 22?

Reply:
For now, this has only been tested on Ubuntu20

Reply:
Update: it also works on Ubuntu 22. I've added NOTE at the beginning.